package ExceptionHandling;

public class Main {
    public static void main(String[] args) {

//        try{
//            System.out.println(5/0);
//        }
//        catch (Exception e){
//            System.out.println("Exception Caught!");
//        }
//
//        System.out.println("Ayberk");

//        System.out.println(Calculator.add(5,5));
//        System.out.println(Calculator.subtract(5,5));
//        System.out.println(Calculator.multiple(5,5));
//        System.out.println(Calculator.divide(5,0));
//        System.out.println(Calculator.divide(5,1));

//        Manager.createStudent("A","A",5);
//        Manager.createStudent("A","A",2);
//        Manager.createStudent("A","A",-45);
//        Manager.createStudent("A","A",5);



    }
}
